ui/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.7.2",
     "@kubernetes/client-node": "github:scality/kubernetes-client-javascript.git#browser-0.10.2-62-g61d8543",
-    "@scality/core-ui": "github:scality/core-ui.git#3865b89",
+    "@scality/core-ui": "github:scality/core-ui.git#ca30cfbcfd1198de6f3667dabd2f3d39b75d1bc0",
```

Comment:
only 7 first characters enough

Reply:
In fact it depends, it's not always 7
Anyway, there is no hurt in writing the whole SHA-1, is it?